src/test/kotlin/game/racingcar/domain/CarTest.kt

```diff
@@ -0,0 +1,71 @@
+package game.racingcar.domain
+
+import game.racingcar.strategy.MoveStrategy
+import game.racingcar.strategy.MustOneMoveStrategy
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+@DisplayName("경주용 자동차")
+internal class CarTest {
+
+    @Test
+    fun `처음 위치는 0`() {
+        // given
+        val car = Car()
+        val expected = 0
+
+        // when
+        // then
+        assertThat(car.position).isEqualTo(expected)
+    }
+
+    @Test
+    fun `1씩 10번 이동`() {
+        // given
+        val car = Car()
+        val expected = 10
+
+        // when
+        for (i in 1..expected) {
```

Comment:
```suggestion
        repeat(10) {
            car.move(MustOneMoveStrategy())
        }
```

`repeat`를 사용하면 좀 명확한 테스트를 할 수 있을 것 같습니다!